modules/oc-adm-by-example-content.adoc

```diff
@@ -729,6 +729,22 @@ Create a new OpenShift release
 == oc adm restart-kubelet
 Restarts kubelet on the specified nodes
 
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Restart all the nodes,  10% at a time
+  oc adm restart-kubelet nodes --all --directive=RemoveKubeletKubeconfig
```

Comment:
All of the other examples have `--parallelism=` does this example require `--parallelism=10%`"?